docs/source/onedal/data-management/graph/directed-adjacency-vector-graph.rst

```diff
@@ -0,0 +1,33 @@
+.. ******************************************************************************
+.. * Copyright 2020-2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _directed_adjacency_vector_graph:
+
+===============================
+Directed adjacency vector graph
+===============================
+
+Class ``directed_adjacency_vector_graph`` is the implementation of directed weighted sparse graph with adjacency matrix underneath 
```

Comment:
class/type is implementation sounds a bit confusing. would need to double-check overall structure of the graph docs/content